src/github/credentials.ts

```diff
@@ -25,6 +28,15 @@ export class CredentialStore {
 		this._configuration = configuration;
 		this._octokits = new Map<string, Octokit>();
 		this._authenticationStatusBarItems = new Map<string, vscode.StatusBarItem>();
+		vscode.commands.registerCommand(AUTH_INPUT_TOKEN_CMD, async () => {
```

Comment:
I'm not sure if adding it to the `package.json` so that it shows up in the command palette makes sense or not

It would be much more clear to have only one command for sign in, but I think it's also good to have a way to directly invoke this

Reply:
Oh, yeah, I thought about doing that, and then didn't do that.

I think it's probably a good idea to be able to invoke it from there. Added.